model/patch/db.go

```diff
@@ -74,6 +75,13 @@ var (
 	TriggerInfoParentPatchKey          = bsonutil.MustHaveTag(TriggerInfo{}, "ParentPatch")
 	TriggerInfoChildPatchesKey         = bsonutil.MustHaveTag(TriggerInfo{}, "ChildPatches")
 	TriggerInfoDownstreamParametersKey = bsonutil.MustHaveTag(TriggerInfo{}, "DownstreamParameters")
+
+	// BSON fields for thirdparty.Github
+	githubPatchHeadOwnerKey       = bsonutil.MustHaveTag(thirdparty.GithubPatch{}, "HeadOwner")
+	githubPatchMergeCommitSHAHKey = bsonutil.MustHaveTag(thirdparty.GithubPatch{}, "MergeCommitSHA")
```

Comment:
maybe the comment should say GithubPatch:
```suggestion
	// BSON fields for thirdparty.GithubPatch
	githubPatchHeadOwnerKey       = bsonutil.MustHaveTag(thirdparty.GithubPatch{}, "HeadOwner")
	githubPatchMergeCommitSHAHKey = bsonutil.MustHaveTag(thirdparty.GithubPatch{}, "MergeCommitSHA")
```